packages/exiftool.vm/exiftool.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>exiftool.vm</id>
-    <version>12.55</version>
+    <version>12.55.0</version>
```

Comment:
This should be `12.55.0.20230121`

Reply:
![2023-01-26_16-14-08_firefox](https://user-images.githubusercontent.com/17606537/214873134-a9fb6431-80ea-4dc7-9c30-555e148e37a5.png)

Reply:
I am not sure we should copy the date from the dependency. If we make a modification in the package we would need to modify the date as well. Our script only adds the date if the version is the same as before the update (when it is strictly necessary): https://github.com/mandiant/VM-Packages/blob/main/scripts/utils/update_package.py#L11

Reply:
Ah, you're right we should just add the current date

Reply:
I have a fix for this. Also looking at the other version issue (#212).